### PR TITLE
Added : signin.html signup .html

### PR DESCRIPTION
**Pull request by Amandeep-sinha**
Please review my PR for signin and signup page

image
image

 you can visit live at : https://optimistic-roentgen-275407.netlify.app/signin.html

### CODE REVIEW

File: signup.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            <a class="top-menu" href="cart.html"><i class="fa fa-shopping-cart fa-2x"><span>2</span></i></a> &nbsp;&nbsp;
+            <a  class="top-menu" href=""><i class="fa fa-heart fa-2x"><span>4</span></i></a>
+            &nbsp;&nbsp;&nbsp;            
+            &nbsp;&nbsp;&nbsp;
+        </div>
+    </nav>
+
+    
+   
+
```
Comment (someshtiwarii) [inline signup.html:34]: remove unnecessary space

File: signin.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            <div class="single">
+                <label for="search" style="color: white;  padding: .5rem; " class="fa fa-search fa-2x"></label>
+                <input type="search" name="search" id="search1" placeholder="Search">
+            </div>
+        
+        <div class="top-navbar-menu">
+            <a href="signin.html" class="top-menu home"><button class="btn">Login</button></a> &nbsp;&nbsp;&nbsp;
+            <a class="top-menu" href="cart.html"><i class="fa fa-shopping-cart fa-2x"><span>2</span></i></a> &nbsp;&nbsp;
+            <a  class="top-menu" href=""><i class="fa fa-heart fa-2x"><span>4</span></i></a>
+            &nbsp;&nbsp;&nbsp;
```
Comment (someshtiwarii) [inline signin.html:27]: avoid using nbsp

File: signin.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+</head>
+<body>
+    <!-- Navbar -->
+    <nav class="top-navbar">
+        <div class="top-navbar-icon">
+            <img src="https://bit.ly/3v2WZoY"/>
+            <div class="title">MetaKart</div>
+        </div>
+            <div class="single">
+                <label for="search" style="color: white;  padding: .5rem; " class="fa fa-search fa-2x"></label>
```
Comment (someshtiwarii) [inline signin.html:19]: avoid inline styling

File: signup.html [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+</head>
+<body>
+    <!-- Navbar -->
+    <nav class="top-navbar">
+        <div class="top-navbar-icon">
+            <img src="https://bit.ly/3v2WZoY"/>
+            <div class="title">MetaKart</div>
+        </div>
+            <div class="single">
+                <label for="search" style="color: white;  padding: .5rem; " class="fa fa-search fa-2x"></label>
```
Comment (someshtiwarii) [inline signup.html:19]: avoid inline styling

File: style.css [hunk 5]
```diff
@@ -0,0 +1,174 @@
+.navigation-bar{
+    gap:60%;
+}
+
+body{
+    background: rgb(224, 220, 220);
```
Comment (someshtiwarii) [inline style.css:6]: use color variables

**Review by someshtiwarii — APPROVED** [review]
make changes to the mentioned stuff, rest looks fine

**Review by suryxks — APPROVED** [review]
+1 to whatever somesh has sugested

**Pull request merged**